### PR TITLE
[windows] [cws] Make FIM processing configurable.

### PR DESCRIPTION
**Pull request by derekwbrown**
For current release, make FIM processing configurable, and disabled by default.

### Motivation

disable feature for initial release

### Describe how to test/QA your changes

Install agent.
Enable CWS
on default config, note that FIM is not enabled.

Enable FIM
```
runtime_security_config:
  fim_enabled: true
  ```
  
Note warning level message in log that it's enabled.

### Reviewer's Checklist
- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [x] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/tests/testopts.go [hunk 1]
```diff
@@ -48,6 +48,8 @@ type testOpts struct {
 	preStartCallback                           func(test *testModule)
 	tagsResolver                               tags.Resolver
 	snapshotRuleMatchHandler                   func(*testModule, *model.Event, *rules.Rule)
+	enableEBPFLess                             bool
```
Comment (safchain) [inline pkg/security/tests/testopts.go:51]: I think this line is not related to this patch

**Review by brycekahle — APPROVED** [review]

**Review by safchain — APPROVED** [review]
